bin/metrics-memcached-graphite.rb

```diff
@@ -54,10 +54,17 @@ class MemcachedGraphite < Sensu::Plugin::Metric::CLI::Graphite
          default: "#{::Socket.gethostname}.memcached"
 
   def run
-    cache = Memcached.new("#{config[:host]}:#{config[:port]}")
+    Timeout.timeout(30) do
```

Comment:
We might want to make this an option, what do you think?

Reply:
IIRC I copied that wholesale from one of the other scripts. I’m totally happy to make it configurable, I can do that tomorrow morning.